client/src/extension.ts

```diff
@@ -83,12 +83,18 @@ export function activate(context: vscode.ExtensionContext) {
           endpoint,
           new AzureKeyCredential(azureApiKey)
         );
-        //result variable will be used in future implementation
         const result = await client.getChatCompletions(
           deploymentId,
           messages,
           parameters
         );
+        const editor = vscode.window.activeTextEditor;
+        editor.edit((editBuilder) => {
+          editBuilder.insert(
+            editor.selection.active,
+            result.choices[0].message.content

```

Comment:
Any chance there may be several choices or there is typically only one? You could also do a sanity check here, if an element in `result.choices` exist.

Reply:
Good point!